db/db.connect.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+// TODO: move to .env/sec
+const initializeDBConnection=async()=>{
+  // Connecting to DB
+  try{
+    await mongoose.connect("mongodb+srv://asohail737:Exits840bulged@neog-cluster.kdlg7.mongodb.net/inventory?retryWrites=true&w=majority", {
```

Comment:
Remove this , Hide this in .env file , might as well use package https://www.npmjs.com/package/dotenv